### DISCUSSION TITLE
Add :not CSS pseudo-class

### DISCUSSION THREAD

**Discussion by seb-jean** [tailwindlabs/tailwindcss#4882] [Ideas]
Hi,

It would be great to have `:not()` CSS pseudo-class:
https://developer.mozilla.org/en-US/docs/Web/CSS/:not

Thanks :)

**Comment by jonathanstanley** [comment]
looking at  https://github.com/tailwindlabs/tailwindcss/blob/167668e9cf275076808c05451e7f193ab47b95dd/src/corePlugins.js

it looks like this might be possible with `createUtilityPlugin` or `addUtilities`?

this could be really nice for lists, for example:`not(first-child)`
